corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -68,7 +68,7 @@ def migrate_from_migration(cls, apps, schema_editor):
                 remaining = cls.count_items_to_be_migrated()
                 if remaining != 0:
                     migrated = False
-                    print("Automatic migration failed, {} items remain to migrate.")
+                    print("Automatic migration failed, {} items remain to migrate.".format(remaining))
```

Comment:
nit: [f-strings](https://www.python.org/dev/peps/pep-0498/) are pretty cool
```suggestion
                    print(f"Automatic migration failed, {remaining} items remain to migrate.")
```